src/styles.ts

```diff
@@ -81,6 +83,9 @@ export const styles = css`
 		font-family: inherit;
 		resize: none;
 	}
+	#input.search {
```

Comment:
This seems wrong. input never has the .search class ?

Reply:
Yes, it's probably a typo/I forgot to delete it